vm/vmimpl/darwin.go

```diff
@@ -0,0 +1,15 @@
+// Copyright 2017 syzkaller project authors. All rights reserved.
+// Use of this source code is governed by Apache 2 LICENSE that can be found in the LICENSE file.
+
+// +build darwin
+
+package vmimpl
+
+import "syscall"
+
+const (
+    CBAUD = 0
```

Comment:
Please prefix these consts with something starting from lower case letter.
Identifiers starting from upper case are exported in Go (part of public package interface), e.g. they will appear in godoc:
http://godoc.org/github.com/google/syzkaller/vm/vmimpl
Something like `unixCBAUD`.